bin/methods/start.js

```diff
@@ -26,14 +30,18 @@ function start (flags, plugins = []) {
   })
 
   // Set defaults
-  if (!config.loader) config.loader = path.join(__dirname, '..', 'loader')
+  if (!config.loader) config.loader = path.join(__dirname, '../../node_modules/cordlr-loader')
   if (plugins.length) {
     if (!config.plugins) config.plugins = []
     config.plugins.push(...plugins)
```

Comment:
I think this should be done near the `Add command-line options to config` comment since this is essentially adding the command line plugins to the config, and not setting defaults. Maybe set the defaults first and then set the command line options